MuxExoPlayer/src/r2_11_1/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -111,6 +112,48 @@ public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName,
     }
   }
 
+  /**
+   * Extracts the tag value from live HLS segment, returns -1 if it is not an HLS stream, not a live
+   * playback.
+   *
+   * @param tagName name of the tag to extract from the HLS manifest.
+   * @return tag value if tag is found and we are playing HLS live stream, -1 string otherwise.
+   */
+  @Override
+  protected String parseHlsManifestTag(String tagName) {
```

Comment:
Can you pull this function implementation up into the `MuxBaseExoPlayer` class and override with the no-op implementation for the player versions where it's not supported?

Reply:
I didn't do this one for time's sake. I'll make a Story to follow up.